mods/cnc/mod.yaml

```diff
@@ -154,6 +154,7 @@ ContentInstaller:
 		.: conquer.mix, desert.mix, general.mix, scores.mix, sounds.mix, temperat.mix, winter.mix
 	ShippedSoundtracks: 2
 	MusicPackageMirrorList: http://www.openra.net/packages/cnc-music-mirrors.txt
+	TFDIndexes: 66, 45, 42, 65, 68, 67, 71, 47, 49, 60, 75, 73
```

Comment:
Adding TFD support for RA and TS should be as simple as defining the appropriate archives in their mod.yaml, right?  Its a bit of a feature creep, but could you please do that too?